src/classad/lexerSource.cpp

```diff
@@ -65,8 +65,8 @@ FileLexerSource::ReadCharacter(void)
 void 
 FileLexerSource::UnreadCharacter(void)
 {
-	//fseek(_file, -1, SEEK_CUR);
-	ungetc(_previous_character, _file);
+	fseek(_file, -1, SEEK_CUR);
```

Comment:
Can you add a comment as to why you use one version over the other?

(I'm sure it's in the ticket, but I forgot already...)